docs/migration_guide.md

```diff
@@ -44,10 +44,19 @@ When upgrading your existing Habor instance to a newer version, you may need to
     ```
 
 7. Unzip the new Harbor package and change to `./harbor` as the working directory. Configure Harbor by modifying the file `harbor.cfg`, 
+
+  - Configure Harbor by modifying the file `harbor.cfg`, 
 you may need to refer to the configuration files you've backed up during step 2. 
 Refer to [Installation & Configuration Guide ](../docs/installation_guide.md) for more information. 
-Since the content and format of `harbor.cfg` may have been changed in the new release, **DO NOT directly copy `harbor.cfg` from previous version of Harbor.**  
+Since the content and format of `harbor.cfg` may have been changed in the new release, **DO NOT directly copy `harbor.cfg` from previous version of Harbor.** 
+
```

Comment:
**IMPORTANT:** If you are upgrading a Harbor instance with LDAP/AD authentication, 
 you must make sure **auth_mode** is set to **ldap_auth** in ```harbor.cfg``` before launching the new version of Harbor. Otherwise, users may not be able to log in after the upgrade.